lua/cfc_time/server/storage_options/sqlite.lua

```diff
@@ -136,7 +136,7 @@ function storage:UpdateBatch( batchData )
 end
 
 function storage:GetTotalTime( steamID, callback )
-    local data = storage:QueryTotalTime( steamID )
+    local data = self:QueryTotalTime( steamID )
     local sum = data[1]["SUM(duration)"]
```

Comment:
Should the whole `[1]["SUM(duration)"]` be done in `QueryTotalTime`? or do we want that to always return the raw query result

Reply:
Mmm, given that this is just used to serve the externally-facing `GetTotalTime` function, I'm thinking it's fine for it to return a dumb response. `QueryTotalTime`'s job is to query the total time, `GetTotalTime`'s job is to use that query result and extract meaning.

There is something to be said about whether or not `GetTotalTime` needs to know/care about the structure of the result, though..

What do you think?